includes/objectcache/MemcachedClient.php

```diff
@@ -1097,11 +1102,14 @@ function _load_items( $sock, &$ret ) {
 				$this->_debugprint( "Error parsing memcached response\n" );
 				// Wikia change - begin
 				// @author macbre (BugId:27916 / PLATFORM-774)
-				$method = explode( '::', wfGetCallerClassMethod( __CLASS__ ) ); // eg. MemcachedPhpBagOStuff::get
+				$method = explode( ':', wfGetCallerClassMethod( __CLASS__ ) ); // eg. MemcachedPhpBagOStuff::get
+				$caller = wfGetCallerClassMethod( [ __CLASS__, 'MemcachedPhpBagOStuff' ] ); // eg. WikiFactory::getWikiByDB
 
 				$this->error( 'MemcachedClient: error parsing the response', [
-					'cmd'       => end( $method ), // eg. get
-					'decl'      => $decl,
+					'caller'    => $caller,
+					'cmd'       => substr( $this->_current_cmd, 0, 1024 ), // e.g. get foo:bar\r\n
```

Comment:
`SET` command can be quite long and we're mainly interested here in getting the key name